Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -680,7 +680,87 @@ def options(opt):
     g.add_option('--ubsan-abort',
         action='store_true',
         help='''Build using the gcc undefined behaviour sanitizer and abort on error''')
-    
+
+
+def write_compilation_database(bld):
+    """
+    Write the clang compilation database as JSON
```

Comment:
clang still relevant here?

Doesn't seem like it, so a comment at least would be good.

Reply:
removed use of clang